generated/nidmm/nidmm_service.cpp

```diff
@@ -1425,8 +1425,7 @@ namespace nidmm_grpc {
       auto status = library_->GetApertureTimeInfo(vi, &aperture_time, &aperture_time_units);
       response->set_status(status);
       if (status == 0) {
-        response->set_aperture_time(static_cast<nidmm_grpc::ApertureTime>(aperture_time));
```

Comment:
Even without your change, I don't see any build failure locally. What am I missing?

Reply:
I am using Microsoft Visual Studio 2017 to compile. It seems the actions only use 2019.

Reply:
Are you making sure to specify the `-A x64` flag for CMake as per the [readme](https://github.com/ni/grpc-device#build-debug)? Between 2017 and 2019, Microsoft changed the default build target from x86 to x64 (hence why we specify it now). That would cause a linker warning since the APIs we link against are x64. I wouldn't be surprised if there were similar issues or differences we hadn't run into yet, though.

Reply:
- I also faced this issue today.
- I use Visual Studio 2019.
- I setup a completely new workspace and ran all the following commands again from scratch but still faced that error being discussed in this PR :
 -- git submodule update --init --recursive
 -- cmake .. -A x64
 -- cmake --build .

-  I built from command prompt and it used the following specifications during the step of `cmake .. -A x64`:
-- Building for: Visual Studio 16 2019
-- Selecting Windows SDK version 10.0.18362.0 to target Windows 10.0.17134.
-- The C compiler identification is MSVC 19.24.28319.0
-- The CXX compiler identification is MSVC 19.24.28319.0

FYI: @doshirohan , @SparkingSpork , @tkrebes

Reply:
@tkrebes , another PR has been submitted to fix this issue. After that PR, issues seems to be resolved on my machine. Can you also check once on your machine and close this PR if it's fixed?